flows/editor.md

```diff
@@ -65,6 +65,24 @@ If we modify the If node's config so the statement is `search.Count < 1` and re-
 
 ![](debug-if-false.png)
 
+## Enabled vs Disabled Nodes
+
+Individual nodes in a flow can be disabled in order to execute parts of a flow without running the entire flow. This feature can be useful for troubleshooting or debugging a flow. For example, one may disable an Email node while troubleshooting a flow in order to avoid sending unnecessary emails.
+
+Nodes are enabled by default. Any node can be disabled or re-enabled through its right-click menu or the configuration pane:
```

Comment:
We also added the ability to disable and re-enable using the toolbar at the top. I think this deserves a mention and an additional arrow in your image.